root/programs/C#/Frameworks/Infrastructure/Framework/Authentication/JwtToken.cs

```diff
@@ -99,16 +101,23 @@ public class JwtToken
                     scopes = JsonConvert.DeserializeObject<List<string>>(jobj["scopes"].ToString());
                 }
 
+                long unixTimeSeconds = 0;
+#if NET45
```

Comment:
@daisukenishino 

Where is this symbol defined?

Reply:
@yusukemaegawa 

See : https://github.com/OpenTouryoProject/OpenTouryo/blob/develop/root/programs/C%23/Frameworks/Infrastructure/Framework/Framework_net45.csproj#L50